docs/source/index.rst

```diff
@@ -2,8 +2,12 @@
    :width: 250 px
 
 GPkit is a Python package for defining and manipulating
-geometric programming (GP) models,
-abstracting away the backend solver.
+geometric programming (GP) models.
+
+The goal of GPkit is to make it easy to create, share, and explore geometric programming models, which tends to align well with being fast and mathematically correct.

```

Comment:
duplicate of text in gp101... and I don't love the above sentence. Maybe move the text below here for more visibility?

```
+Our hopes are to bring the mathematics of Geometric Programming
+into the engineering design process
+in a disciplined and collaborative way, and to
+encourage research with and on GPs by providing an
+easily extensible object-oriented framework.
```